pkg/cmd/volume/ls.go

```diff
@@ -44,22 +44,76 @@ type volumePrintable struct {
 }
 
 func Ls(options *types.VolumeLsCommandOptions, stdout io.Writer) error {
```

Comment:
I think we can keep the input and output logic within the original `Ls` function, and only split the core logic out to a new `List` function (e.g., don't need to put I/O in separate `SanitizeInput` and `PrintOutput` funcs. This will minimize the diff and keep its original format. Other callers will just reuse the `List` func, not `Ls` anyway.

This way, we're only doing a small refactor that extracts a core logic out for reusability while keep nerdctl as it is (e.g., `Ls` only need to call the new `List` when needed instead of splitted into 3 sub function calls.